src/tools/ScenarioMeasurement/Startup/ProcessTimeParser.cs

```diff
@@ -17,7 +17,7 @@ public void EnableKernelProvider(TraceEventSession kernel)
 
         public void EnableUserProviders(TraceEventSession user)
         {
-            user.EnableProvider("PerfLabProcessTimeEventSource");
+            return;
```

Comment:
because this method returns void, you don't need a return statement here. It can be an empty method body.